benchmarks/decoders/benchmark_decoders_library.py

```diff
@@ -419,16 +428,16 @@ def plot_data(df_data, plot_path):
             ax = axes[row, col]  # Select the appropriate axis
 
             # Set the title for the subplot
-            base_video = os.path.basename(video)
+            base_video = os.path.basename(video).removesuffix(".mp4")
```

Comment:
We should stop using `os.path`, I had tried to migrate awa from those already in https://github.com/pytorch/torchcodec/pull/249

I know this wasn't introduced in this PR but do you mind relyin on `pathlib` for this?